db/migrate/20220912100910_create_movies.rb

```diff
@@ -0,0 +1,11 @@
+class CreateMovies < ActiveRecord::Migration[7.0]
+  def change
+    create_table :movies do |t|
+      t.string :name
+      t.text :description
+      t.integer :duration
```

Comment:
You can consider using [interval data type](https://www.bigbinary.com/blog/rails-6-1-adds-postgresql-interval-data-type) 🤔